database/redis/metric_test.go

```diff
@@ -443,6 +443,7 @@ func TestMetricSubscription(t *testing.T) {
 		So(err, ShouldBeNil)
 		err = dataBase.SaveMetrics(map[string]*moira.MatchedMetric{metric2: met2})
 		So(err, ShouldBeNil)
+		<-time.After(time.Second * 6)
```

Comment:
```suggestion
		time.Sleep(time.Second * 6)
```

Reply:
I suggest to avoid channels if there is no need

Reply:
And question: Why this test passes only if use 6 sec delay? I try to run without delay and it fails, but why? And why it not fails before "n checkers" feature?

Reply:
Before this feature polling was based on PubSub subscription and kill() operation within Tomb sent unsubscribe message to redis. That model gave time while unsubscribe message processing to finish other queue messages. Now killing happens immediately

Reply:
You are right